packages/org-unit-dialog/src/OrgUnitSelector.js

```diff
@@ -173,26 +192,39 @@ class OrgUnitSelector extends Component {
                             selected={this.props.selected}
                             styles={styles.userOrgUnits}
                             userOrgUnits={this.props.userOrgUnits}
-                            handleUserOrgUnitClick={this.props.handleUserOrgUnitClick}
+                            handleUserOrgUnitClick={
+                                this.props.handleUserOrgUnitClick
+                            }
                             checkboxColor={this.props.checkboxColor}
                         />
-                        <div style={styles.scrollableContainer.overlayContainer}>
+                        <div
+                            style={styles.scrollableContainer.overlayContainer}
+                        >
                             {this.props.userOrgUnits.length > 0 && (
-                                <div style={styles.scrollableContainer.overlay} />
+                                <div
+                                    style={styles.scrollableContainer.overlay}
+                                />
                             )}
-                            <OrgUnitTree
+                            <OrgUnitTreeMultipleRoots
+                                roots={this.props.roots}
                                 root={this.props.root}
-                                selected={this.props.selected.map(orgUnit => orgUnit.path)}
+                                selected={this.props.selected.map(
+                                    orgUnit => orgUnit.path
+                                )}
                                 initiallyExpanded={this.state.initiallyExpanded}
                                 onSelectClick={this.props.handleOrgUnitClick}
                                 onExpand={this.onExpand}
                                 onCollapse={this.onCollapse}
                                 onContextMenuClick={this.onContextMenuClick}
                                 treeStyle={styles.orgUnitTree.treeStyle}
                                 labelStyle={styles.orgUnitTree.labelStyle}
-                                selectedLabelStyle={styles.orgUnitTree.selectedLabelStyle}
+                                selectedLabelStyle={
+                                    styles.orgUnitTree.selectedLabelStyle
+                                }
                                 checkboxColor={this.props.checkboxColor}
-                                displayNameProperty={this.props.displayNameProperty}
+                                displayNameProperty={
+                                    this.props.displayNameProperty
+                                }
                                 showFolderIcon
                                 disableSpacer
                             />
```

Comment:
This is the only functional change from what I can see. Is that correct?

Reply:
Sorry, after the last commit I lost reference here.
I believe you are referring to the addition of the {{roots}} prop, yes, that is the change.
I kept the prettier commit separated, so it's easier to see the actual change in commit: https://github.com/dhis2/d2-ui/pull/446/commits/936c2ba2bb9eff65824a247bf37224236f7d9bfa